docs/commands/clip.md

```diff
@@ -0,0 +1,15 @@
+# clip
+
+Copy the contents of the pipeline to the copy/paste buffer
+
+
+# Syntax
+```cmd
+<command> clip
+```
+
+## Examples
+1. To copy the current directory list:
+```cmd
+ls clip
```

Comment:
You are doing `> ls clip`, `ls` lists the contents of what you are passing into it, in this case, the path `clip`. This is not correct. `ls` and `clip` are different commands and you are not using `clip` here.